components/infobox/wikis/deadlock/infobox_item_custom.lua

```diff
@@ -0,0 +1,231 @@
+			{name = 'Bullet Shield Health', parameter = 'bulletshield'},
+			{name = 'Bullet Resist', parameter = 'bulletresist', funct = '_positivePercentDisplay'},
+			{name = 'Bullet Resist vs. NPCs', parameter = 'bulletresistnpc', funct = '_positivePercentDisplay'},
+			{name = 'Sprint Speed', parameter = 'sprintspeed'},
+			{name = 'Stamina', parameter = 'Stamina'},
+			{name = 'Slide Distance', parameter = 'slidedistance', funct = '_positivePercentDisplay'},
+			{name = 'Spirit Power', parameter = 'spiritpower'},
+			{name = 'Spirit Resist', parameter = 'spiritresist'},
+			{name = 'Spirit Shield Health', parameter = 'spiritshield'},
+			{name = 'Spirit Lifesteal', parameter = 'spiritlifesteal', funct = '_positivePercentDisplay'},
+			{name = 'Cooldown Reduction', parameter = 'cdreduction', funct = '_positivePercentDisplay'},
+			{name = 'Movement Speed', parameter = 'movespeed'},
+		}
+		widgets = caller:_getAttributeCells(attributeCells)
+		table.insert(widgets, Cell{name = 'Standard Bonus', children = {args.standardbonus}})
+		if not Table.isEmpty(widgets) then
+			table.insert(widgets, 1, Title{children = 'Attributes'})
+		end
+		return widgets
+	elseif id == 'ability' then
+		if String.isEmpty(args.use) and String.isEmpty(args.active) and String.isEmpty(args.passive) then
+			return {}
+		end
+		Array.appendWith(widgets,
+			Cell{name = 'Active', children = {args.active, args.active2, args.active3}},
+			Cell{name = 'Passive', children = {args.passive, args.passive2, args.passive3}}
+		)
+	elseif id == 'availability' then
+		if String.isEmpty(args.category) and String.isEmpty(args.drop) then return {} end
+		return {
+			Title{children = 'Type'},
+			Cell{name = 'Category', children = {caller:_categoryDisplay()}},
+			Cell{name = 'Tier', children = {args.tier}},
+		}
+	elseif id == 'recipe' then
+		if String.isEmpty(args.recipe) then return {} end
+		return {
+			Title{children = 'Components'},
+			Center{children = {args.recipe}}
+		}
+	elseif id == 'maps' then return {}
+	elseif id == 'info' then return {}
+	end
+
+	return widgets
+end
+
+---@param args table
+---@return string[]
+function CustomItem:getWikiCategories(args)
+	if not Namespace.isMain() then return {} end
+
+	local categories = {}
+	if String.isNotEmpty(args.str) or String.isNotEmpty(args.agi) or String.isNotEmpty(args.int) then
+		table.insert(categories, 'Attribute Items')
+	end
+
+	if String.isNotEmpty(args.movespeed) or String.isNotEmpty(args.movespeedmult) then
+		table.insert(categories, 'Movement Speed Items')
+	end
+
+	local possibleCategories = {
+		['Items with Active Abilities'] = 'active',
+		['Items with Passive Abilities'] = 'passive',
+	}
+	for category, requiredArg in pairs(possibleCategories) do
+		if String.isNotEmpty(args[requiredArg]) then
+			table.insert(categories, category)
+		end
+	end
+	return categories
+end
+
+---@param args table
+---@return string?
+function CustomItem.nameDisplay(args)
+	return args.itemname
+end
+
```

Comment:
```suggestion
```

And bot change `itemname` to `name` on the wiki

Also update the other uses in here of `itemname` to `name`